tests/disk/disk.go

```diff
@@ -30,7 +32,8 @@ import (
 
 const (
 	// DiskImageSize is the default file size(1GB) used while creating backing image
-	DiskImageSize = 1073741824
+	DiskImageSize       = 1073741824
+	DiskImageNamePrefix = "openebs-disk-xfs_quota"
```

Comment:
Does the disk image name always has the xfs_quota in it?

Reply:
Didn't get the question.

Reply:
The image name is hardcoded with xfs_quota. May be it can be like
```suggestion
	DiskImageNamePrefix = "openebs-disk"
	XFSQuotaDiskImageName = DiskImageNamePrefix + "-" +  "xfs_quota"
```

Reply:
The entire thing is prefixed to a hash generated by tempfile so that the name is unique.

Reply:
ok. then rename it to `XFSQuotaDiskImageNamePrefix` or something like that, because just the DiskIMageNamePrefix is too generic.